src/signals/incident-management/containers/ReporterContainer/ReporterContainer.tsx

```diff
@@ -4,6 +4,7 @@ import React from 'react'
 import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
 import styled from 'styled-components'
 import LoadingIndicator from 'components/LoadingIndicator'
+import Pagination from 'components/Pagination'
```

Comment:
You're able to use [the ASC](https://amsterdam.github.io/amsterdam-styled-components/?path=/story/ui-compactpager--default-story) component instead (this wasn't available earlier)